remote-trait-object-tests/src/test_concurrent_ping.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 Kodebox, Inc.
```

Comment:
There is an empty file `remote-trait-object-tests/src/test_concurrent_ping/mod_ping.rs`

Reply:
Thanks. I removed it.